framework/calico_utils.py

```diff
@@ -0,0 +1,33 @@
+import os
+import logging
+import logging.handlers
+import errno
+
+
+def _setup_logging(logfile):
+    _log = logging.getLogger(__name__)
```

Comment:
In this file `__name__` is going to be `"calico_utils"`, which probably isn't what you want.  How about just hardcoding it to `"calico_framework"`?